cli/command/telemetry_docker.go

```diff
@@ -0,0 +1,121 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"path"
+
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+const otelConfigFieldName = "otel"
+
+// dockerExporterOtlpEndpoint retrieves the OTLP endpoint used for the docker reporter
+// from the current context.
+func dockerExporterOtlpEndpoint(cli Cli) (endpoint string, secure bool, err error) {
```

Comment:
isn't OTLP an acronym? in Go acronyms are always all caps

 ```suggestion
func dockerExporterOTLPEndpoint(cli Cli) (endpoint string, secure bool, err error) {
```